File: WEKEE.FE/UI.HOME/src/scenes/Admin/APermission/dto/atomicReadDto.ts
export interface AtomicReadDto {
    id: number;
    name: string;
    typesRsc: number;
    description: string;
    isActive: boolean | null;
    createBy: number;
    createByName: string;
    createdOnUtc: string;
    updatedOnUtc: string;
}

File: WEKEE.BE/Account.Domain/Shared/DataTransfer/AtomicDTO/AtomicReadDto.cs
using Account.Domain.Shared.Common;
using System;
using System.Collections.Generic;
using System.Text;

namespace Account.Domain.Shared.DataTransfer.AtomicDTO
{
    public class AtomicReadDto:EntityKey<int>
    {
        public string Name { get; set; }
        public int TypesRsc { get; set; }
        public string Description { get; set; }
        public bool? IsActive { get; set; }
        public int CreateBy { get; set; }
        public string CreateByName { get; set; }
        public DateTime CreatedOnUtc { get; set; }
        public DateTime UpdatedOnUtc { get; set; }
    }
}
